source/mainnet/net/installation/downloads-testnet.rst

```diff
@@ -85,6 +85,14 @@ Concordium Client v4.1.0
 
    - SHA256 checksum of the download: |client-linux-checksum|
 
+   |
+
+   When you run Concordium Client on Linux, run the following command to make the file you download executable:
+
+      .. code-block:: console
+
+         chmod +x concordium-client
```

Comment:
The file that they download will be called (at the moment) `concordium-client_4.1.0` so technically they should run
```
chmod +x concordium-client_4.1.0
```

But it is not a good idea to write the exact version number since it'll be hard to maintain. We could write instead
```
chmod +x concordium-client*
```